setup-meet.yml

```diff
@@ -0,0 +1,16 @@
+- hosts: meet.adfc-intern.de
+  gather_facts: no
+  roles:
+    - proxmox-container
+  tasks:
+    - name: Installiere Pakete
+      apt:
+        name:
+          - python-apt
+          - htop
+- hosts: meet.adfc-intern.de
+  gather_facts: no
+  roles:
+    - ssh-keys
+    - geerlingguy.certbot
```

Comment:
# Ich finde in der role diverse Hinweise bezüglich TURN-Server, die ich haber nicht durchschaue. IDatenschutztechnisch wäre es vielleicht schöner, eigen STUND/TURN-Server zu verwenden (Kuketz macht das auch mit eigenem Server)- oder wenigstens welche, die nicht bon google betrieben werden.
 Wie ist der Status jetzt?

Reply:
Keine Ahnung. Das müssten wir noch evaluieren.

Reply:
Hab was in der Datei turn meet.adfc-intern.de-config.js gefunden, ist aber nur bei 2 Teilnehmern (P2p_mode) relevant)
  // The STUN servers that will be used in the peer to peer connections
        stunServers: [

            { urls: 'stun:meet-jit-si-turnrelay.jitsi.net:443' },
Solange das läuft brauchen wir das ja nicht ändern. Aber datenschutztechnisch werden wir da bei Dauerbetrieb ran müssen, oder?

Reply:
Wir brauchen auf jeden Fall noch einen lokalen STUN/TURN (z. B. coturn), weil wir das sonst nicht DSGVO-konform betreiben können.

Reply:
Jup installieren wir dazu